### ISSUE TITLE
The baby zombie model becomes a normal zombie

### ISSUE THREAD

**Issue by BaiGave** [Sam54123/mc-world-export#17]
I found that the model of the baby zombie has changed to that of the norma zombie, please fix this bug

**Comment by Sam54123** [comment]
While that's on the todo list, and the exact reason behind it *is* known (and rather straightforward), it's not a priority rn. Do you have a particular project that it's causing issues with?

**Comment by BaiGave** [comment]
No, but at the same time, I would like to ask, the entities in the mod seem to be unable to display too, is there any way to solve this problem？

**Comment by Sam54123** [comment]
Which entities in particular? Due to the way Minecraft's rendering engine is structured, support for each entity must be [coded manually](https://github.com/Sam54123/mc-world-export/blob/main/doc/entity_export_design_doc.md). Not all entities are supported atm, and entities that are not supported will come at a later date.

**Comment by BaiGave** [comment]
Then in the future, will there be a function for us to add mod creatures? After all, there are too many mod creatures, and it is too tiring for one person to add them manually.
